test/dropRepeatsWith.js

```diff
@@ -0,0 +1,41 @@
+var assert = require('assert');
+
+var R = require('..');
+
+
+describe('dropRepeatsWith', function() {
+  var objs = [
+    {i: 1}, {i: 2}, {i: 3}, {i: 4}, {i: 5}, {i: 3}
+  ];
+  var objs2 = [
+    {i: 1}, {i: 1}, {i: 1}, {i: 2}, {i: 3},
+    {i: 3}, {i: 4}, {i: 4}, {i: 5}, {i: 3},
+  ];
+  function eqI(x, accX) { return x.i === accX.i; }
```

Comment:
I don't recall the history.  But it's quite possible that `uniqWith` predates `eqProps`.  It would be good to change that one too, I suppose.